Dockerfile

```diff
@@ -35,9 +35,13 @@ jdk.unsupported\
 # We extract JRE's hard dependencies, libz and SSL certs, from the fat JRE image.
 FROM gcr.io/distroless/java:11-debug AS deps
 
+# Mainly, this gets BusyBox
 FROM gcr.io/distroless/cc:debug
 
+# Similar to Alpine Linux, we ensure /bin/sh works (via BusyBox)
 SHELL ["/busybox/sh", "-c"]
```

Comment:
Think we can delete this

Reply:
hehe we need it :) we need to use busybox to execute the other command

Reply:
found another way https://github.com/openzipkin/docker-jre-full/pull/21